src/ontology/mondo.Makefile

```diff
@@ -696,6 +697,21 @@ update-ordo-subsets:
 	$(ROBOT) merge -i $(SRC) -i $(TMPDIR)/external/processed-ordo-subsets.robot.owl --collapse-import-closure false convert -f obo --check false -o $(SRC).obo
 	mv $(SRC).obo $(SRC) && make NORM && mv NORM $(SRC)
 	
+####################################
+##### MalaCards #####################
+####################################
+
+.PHONY: update-malacards
+update-malacards:
+	# TODO BEFOR MERGING replace this next download line with the commented line right afterwards.
```

Comment:
@twhetzel draft because of this. The Mondo ingest pr on malacards is ready

Reply:
Ok, that one was Draft too, but I approved it.